scss/civicrm/contact/_tabs.scss

```diff
@@ -11,11 +13,11 @@
       border-radius: 0;
       box-shadow: $box-shadow;
       float: left;
-      padding: 0 0 99999px;
       margin-bottom: -99999px;
+      padding: 0 0 99999px;
```

Comment:
why this value?

Reply:
I haven't changed any on this file, this is stylelint change, so, I have just added header of `stylelint-disable`